@xen-orchestra/web/src/components/backups/logs/panel/cards/BackupLogInfosCard.vue

```diff
@@ -6,64 +6,36 @@
       </UiLink>
     </UiCardTitle>
     <div class="content">
-      <VtsCardRowKeyValue>
-        <template #key>{{ t('id') }}</template>
-        <template #value>{{ backupLog.id }}</template>
-        <template #addons>
-          <VtsCopyButton :value="backupLog.id" />
-        </template>
-      </VtsCardRowKeyValue>
-      <VtsCardRowKeyValue>
-        <template #key>{{ t('start-date') }}</template>
-        <template #value>{{ formattedStartDate }}</template>
-        <template v-if="formattedStartDate !== undefined" #addons>
-          <VtsCopyButton :value="formattedStartDate" />
-        </template>
-      </VtsCardRowKeyValue>
-      <VtsCardRowKeyValue>
-        <template #key>{{ t('end-date') }}</template>
-        <template #value>{{ formattedEndDate }}</template>
-        <template v-if="formattedEndDate !== undefined" #addons>
-          <VtsCopyButton :value="formattedEndDate" />
-        </template>
-      </VtsCardRowKeyValue>
-      <VtsCardRowKeyValue>
-        <template #key>{{ t('duration') }}</template>
-        <template #value>{{ duration }}</template>
-        <template v-if="duration !== undefined" #addons>
-          <VtsCopyButton :value="duration" />
-        </template>
-      </VtsCardRowKeyValue>
-      <VtsCardRowKeyValue>
-        <template #key>{{ t('status') }}</template>
+      <UiLabelValue :label="t('id')" :value="backupLog.id" :copy-value="backupLog.id" ellipsis />
+      <UiLabelValue :label="t('start-date')" :value="formattedStartDate" :copy-value="formattedStartDate" ellipsis />
+      <UiLabelValue :label="t('end-date')" :value="formattedEndDate" :copy-value="formattedEndDate" ellipsis />
+      <UiLabelValue :label="t('duration')" :value="duration" :copy-value="duration" ellipsis />
```

Comment:
```suggestion
      <UiLabelValue :label="t('id')" :value="backupLog.id" :copy-value="backupLog.id" />
      <UiLabelValue :label="t('start-date')" :value="formattedStartDate" :copy-value="formattedStartDate" />
      <UiLabelValue :label="t('end-date')" :value="formattedEndDate" :copy-value="formattedEndDate" />
      <UiLabelValue :label="t('duration')" :value="duration" :copy-value="duration" />
```
<img width="418" height="570" alt="image" src="https://github.com/user-attachments/assets/e22b7dc2-bd8d-485a-9e47-8eda5400cae5" />

<img width="454" height="285" alt="image" src="https://github.com/user-attachments/assets/c9d4829f-5523-464a-bca7-707595bf1022" />